pkg/controller/masteruserrecord/sync_test.go

```diff
@@ -243,8 +300,48 @@ func TestSyncMurStatusWithUserAccountStatusWhenCompleted(t *testing.T) {
 
 	uatest.Modify(userAccount, uatest.StatusCondition(toBeProvisioned()))
 
-	// when and then
-	testSyncMurStatusWithUserAccountStatus(t, userAccount, mur, scheme, toBeProvisioned(), toBeProvisionedNotificationCreated())
+	t.Run("successful", func(t *testing.T) {
+		// given
+		hostClient := test.NewFakeClient(t, mur)
+		sync, memberClient := prepareSynchronizer(t, userAccount, mur, hostClient)
+
+		// when
+		err := sync.synchronizeStatus()
+
+		// then
+		require.NoError(t, err)
+		verifySyncMurStatusWithUserAccountStatus(t, memberClient, hostClient, userAccount, mur, toBeProvisionedNotificationCreated(), toBeProvisionedNotificationCreated())
```

Comment:
just curious, why to you need `toBeProvisionedNotificationCreated()` twice?

Reply:
wrong copy-paste :man_facepalming: 
good catch, thanks - fixed in 2368e4f